changelogs/fragments/47492-docker_network-add-ipv6-support.yaml

```diff
@@ -0,0 +1,6 @@
+minor_changes:
+  - "docker_network - Add support for ipv6 networks."
+  - "docker_network - Minimum docker-py version increased from ``1.9.0`` to ``1.10.0``."
```

Comment:
Yes, that's probably the best way. Or create a new changelog fragment `docker_network-requirements.yaml` which contains that one, and remove it from both yours and the other changelog fragment. In that case, there's one unique place where it can be bumped in the future.

Reply:
merged it into the newly proposed file name